src/h5web/App.tsx

```diff
@@ -32,32 +38,40 @@ function App(): JSX.Element {
 
         <ReflexSplitter />
 
-        <ReflexElement minSize={500}>
-          <ReflexContainer orientation="horizontal">
-            <ReflexElement className={styles.dataVisualizer} minSize={250}>
-              <DatasetVisualizer dataset={selectedDataset} />
-            </ReflexElement>
-
-            <ReflexSplitter />
-
-            <ReflexElement
-              className={styles.metadataViewer}
-              flex={0.25}
-              minSize={100}
+        <ReflexElement className={styles.mainArea} minSize={500}>
+          <div className={styles.roleToggler}>
+            <button
+              type="button"
+              role="tab"
+              className={styles.btn}
+              aria-selected={role === Role.Inspect}
+              onClick={() => {
+                setRole(Role.Inspect);
+              }}
+            >
+              Inspect
+            </button>
+            <button
+              type="button"
+              role="tab"
+              className={styles.btn}
+              aria-selected={role === Role.Visualize}
+              onClick={() => {
+                setRole(Role.Visualize);
+              }}
             >
-              {selectedLink ? (
-                <MetadataViewer
-                  key={JSON.stringify(selectedLink)}
-                  link={selectedLink}
-                  entity={selectedEntity}
-                />
-              ) : (
-                <div className={styles.empty}>
-                  <p>No entity selected.</p>
-                </div>
```

Comment:
The logic for displaying an empty div was moved inside `MetaViewer`. This was easier to implement the role toggler and consistent with the behaviour of `DatasetVisualizer`